experiments/lammps/rocm/ramble.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ramble:
+  include:
+  - ./configs/spack.yaml
+  - ./configs/variables.yaml
+  - ./configs/modifier.yaml
+
+  config:
+    deprecated: true
+    spack_flags:
+      install: '--add --keep-stage'
+      concretize: '-U -f'
+
+  modifiers:
+  - name: allocation
+
+  applications:
+    lammps:
+      workloads:
+        hns-reaxff:
+          env_vars:
+            set:
+              LD_LIBRARY_PATH: '/opt/cray/pe/cce/16.0.0/cce/x86_64/lib:$LD_LIBRARY_PATH'
+              MPICH_GPU_SUPPORT_ENABLED: 0
```

Comment:
@pearce8 @scheibelp The code works on tioga, but I could not get around having to hard code `LD_LIBRARY_PATH` and `MPICH_GPU_SUPPORT_ENABLED` directly in `ramble.yaml`. Is it ok if we merge the PR right now and tackle this issue in a separate PR?